src/serac/physics/heat_transfer.hpp

```diff
@@ -224,47 +250,54 @@ class HeatTransfer<order, dim, Parameters<parameter_space...>, std::integer_sequ
   }
 
   /**
-   * @brief Set the thermal flux and mass properties for the physics module
+   * @brief Set the thermal flux and heat capacity properties for the physics module
```

Comment:
The wording of this is a little misleading. The heat flux and the heat capacity are *outputs* of the material model, not properties to set. Maybe make this simpler, like "Set the material model for this region" or something like that.